.github/workflows/metallb_e2e.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           repository: metallb/metallb
           path: metallb
-          ref: a4b2482c334678ae79d79b5f5b3196ad760b48ac
+          ref: 7df916e921fcf247bd017834a2a9a85aa0a6be03
```

Comment:
I probably asked the this question at some point in the past, but why are we pinning to a commit id? Why not set this to master to always be testing against the latest code?

Reply:
In general, we want to tie ourselves to manifests from a specific version/point in time and support them, specifying master directly could be misleading - which "version" of the master are we currently working with?
If we're keeping that approach, we should also run the e2e tests from the same MetalLB version that we deploy using the Operator (and maybe master e2e added stuff that the Operator doesn't support yet).